waltz-tools/src/test/java/com/wepay/waltz/tools/performance/PerformanceCliTest.java

```diff
@@ -122,6 +122,23 @@ public void testRunConsumer() throws Exception {
 
             String expectedCmdOutput = "Read 10 transactions";
             assertTrue(outContent.toString("UTF-8").contains(expectedCmdOutput));
+
+            String[] argsMountLatest = {
+                    "test-consumers",
+                    "--txn-size",
+                    "128",
+                    "--num-txn",
+                    "10",
+                    "--cli-config-path",
+                    configFilePath,
+                    "--num-active-partitions",
+                    "1",
+                    "--mount_from_latest"
+            };
+            PerformanceCli.testMain(argsMountLatest);
+
+            assertTrue(outContent.toString("UTF-8").contains(expectedCmdOutput));
```

Comment:
Make sure to reset the outContent (i.e. outContent.reset()) before line 138. Otherwise, this might be asserting based on line 124.